File: nbui-pull/src/main/java/cn/rubintry/nbui/pull/ElasticPullFactory.java
package cn.rubintry.nbui.pull;

public class ElasticPullFactory {
    /**
     * 弹性阻力系数
     */
    public static ElasticPull create(Double elasticCoefficient) {
        return new ElasticPull(elasticCoefficient);
    }
}

File: nbui-pull/src/main/java/cn/rubintry/nbui/pull/ElasticPullFactory.kt
package cn.rubintry.nbui.pull

internal object ElasticPullFactory {

    /**
     * 弹性阻力系数
     */
    @JvmStatic
    fun create(elasticCoefficient: Double): ElasticPull{
        return ElasticPull(elasticCoefficient)
    }
}